### DISCUSSION TITLE
differential test for atacseq data alone

### DISCUSSION THREAD

**Discussion by dedeepyavaka** [stuart-lab/signac#1722] [Q&A]
Hi, 

We would like to have a Differential comparison for the peaks between our timept data. We are primarily interested in peaks that are differential in atacseq but not in transcriptomic data from rnaseq. For now, we only have atac seq data as our rna seq did not perform well. Is there a way to get differential peaks for atac seq data alone.

Thanks,

**Comment by timoast** [comment]
Hi, some examples of how to do this are provided on the Signac website: https://stuartlab.org/signac/articles/pbmc_vignette#find-differentially-accessible-peaks-between-cell-types

You can use the `FindMarkers()` function
